planner/cascades/optimize.go

```diff
@@ -25,24 +25,34 @@ import (
 
 // FindBestPlan is the optimization entrance of the cascades planner. The
 // optimization is composed of 2 phases: exploration and implementation.
-func FindBestPlan(sctx sessionctx.Context, logical plannercore.LogicalPlan) (plannercore.Plan, error) {
+func FindBestPlan(sctx sessionctx.Context, logical plannercore.LogicalPlan) (p plannercore.PhysicalPlan, err error) {
```

Comment:
I prefer to keep using `Plan` as the return value since we may return `Insert`/`Update`/`Delete` plan.